tests/testthat/test-who_palettes.R

```diff
@@ -0,0 +1,10 @@
+context("who_palettes")
+
+test_that("who_palettes cannot take n larger than palettes or theme not provided", {
+  
+  large_n <- who_palettes(palette = "main", n = 100)
+  expect_error(large_n, "n is larger than number of available colours for chosen palette")4
```

Comment:
Seeing type at the end of the line (4) causing this to fail.

Reply:
I removed the 4 but that doesn't seem to have solved the problem